components/constellation/pipeline.rs

```diff
@@ -119,6 +120,8 @@ pub struct InitialPipelineState {
     pub webrender_api_sender: webrender_traits::RenderApiSender,
     /// Whether this pipeline is considered private.
     pub is_private: bool,
+    /// A channel to the webvr thread.
+    pub webvr_thread: Option<IpcSender<WebVRMsg>>,
```

Comment:
It's used to create the ScriptThreadFactory. When a script thread is created it gets a cloned handle to the webvr_thread sender. This sender  is used by DOMObjects to comunicate with the WebVRThread and send messages like: getDisplay, requestPresent, etc.